gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Wed Jun 06 15:50:00 BST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
```

Comment:
what's this change for?

Reply:
I've always thought it was weird we used 'all' rather than just 'bin' so this looks like a good change to me!

Reply:
I don't know, gradle must've made this change for me...I'll change it back seeing as I didn't intend to do it

Reply:
In which case you'll need to revert `gradlew.bat` as well